test/test_fx.py

```diff
@@ -714,6 +715,17 @@ def test_wrong_topo(self):
         nodes[2], nodes[3] = nodes[3], nodes[2]
         with self.assertRaisesRegex(RuntimeError, 'was used before it has been defined'):
             graph.lint()
+    def test_find_single_partition(self):
+        class testModule(torch.nn.Module):
+            def forward(self, a, b):
+                return a + b
+        m = testModule()
+        traced = symbolic_trace(m)
+        partitioner = Partitioner.Partitioner()
+        devices = [{"device_0": float('inf')}]
```

Comment:
nit: it might be more clear if the input was [{"name":"dev0,"available_mem":float('inf')}] This way if we need to add another named field to device later if we need to.